frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts

```diff
@@ -134,6 +134,22 @@ function snapshotDescription(snapshot: eventWithTime): string {
     return snapshotTypeName + suffix
 }
 
+function timeRelativeToStart(
+    thingWithTime: eventWithTime | PerformanceEvent | RecordingConsoleLogV2 | RecordingEventType,
+    start: Dayjs | undefined
+): {
+    timeInRecording: number
+    timestamp: dayjs.Dayjs
+} {
+    const timestamp = dayjs(thingWithTime.timestamp)
+    const timeInRecording = timestamp.valueOf() - (start?.valueOf() ?? 0)
+    if (timeInRecording < 0) {
+        // eslint-disable-next-line no-console
+        console.log('[wat] Negative time in recording', { timeInRecording, timestamp, start, thingWithTime })
```

Comment:
Should this be a `console.error`?

Reply:
Only if we generate those timestamps would be an error, if we read from somewhere, nothing we can do anyway.

Reply:
ah, no, leftover from the draft PR 🙈